aws/metaData.go

```diff
@@ -0,0 +1,63 @@
+package aws
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+func GetMetaData(path string) (contents []byte, err error) {
+	url := "http://169.254.169.254/latest/meta-data/" + path
```

Comment:
Shouldn't `/meta-data` be specified by the user in `path` since the [`user-data` and `dynamic` endpoints](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html) are also hosted?